normalize/record/json.py

```diff
@@ -239,7 +240,7 @@ def to_json(record, extraneous=True, prop=None):
     elif isinstance(record, (list, tuple, set, frozenset)):
         return list(_json_data(x, extraneous) for x in record)
 
-    elif isinstance(record, (basestring, int, float, type(None))):
+    elif isinstance(record, (basestring, int, float, type(None), decimal.Decimal)):
```

Comment:
Should be able to add a decimal value to a new or existing test to memorialize the functionality.